scene/gui/graph_node.cpp

```diff
@@ -459,7 +459,7 @@ void GraphNode::_shape() {
 }
 
 void GraphNode::set_slot(int p_idx, bool p_enable_left, int p_type_left, const Color &p_color_left, bool p_enable_right, int p_type_right, const Color &p_color_right, const Ref<Texture2D> &p_custom_left, const Ref<Texture2D> &p_custom_right) {
-	ERR_FAIL_COND(p_idx < 0);
+	ERR_FAIL_COND_MSG(p_idx < 0, vformat("Cannot set because Slot ID p_idx = %d is smaller than 0.", p_idx));
```

Comment:
```suggestion
	ERR_FAIL_COND_MSG(p_idx < 0, vformat("Cannot set slot with p_idx (%d) lesser than zero.", p_idx));
```

Reply:
![question](https://user-images.githubusercontent.com/17506575/118123186-9e692000-b413-11eb-80e7-dca8d1909ef2.png)

Any reason why slot_enable_left/right use one format for showing p_idx,
while rest others use another format?